docs/index.asciidoc

```diff
@@ -0,0 +1,29 @@
+:plugin: elastic_enterprise_search
+:type: integration
+:no_codec:
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: %VERSION%
+:release_date: %RELEASE_DATE%
+:changelog_url: %CHANGELOG_URL%
+:include_path: ../../../../logstash/docs/include
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="plugins-{type}s-{plugin}"]
+
+=== Elastic EnterpriseSearch Integration Plugin
```

Comment:
```suggestion
=== Elastic Enterprise Search integration plugin
```

Reply:
Enterprise Search should always be two words.  The capitalization you used is consistent with other integration plugins, but we'll be changing it to be consistent with other plugins. This is a good place to start. 😄